server/svix-server/src/v1/utils/mod.rs

```diff
@@ -175,6 +175,34 @@ pub fn apply_pagination<
     }
 }
 
+/// Applies sorting and filtration to a query from its iterator, sort column, and limit,
+/// in an ascending ordering, i.e. one where `Normal` corresponds to starting at the lowest
+/// and counting up, and `Prev` corresponds to starting at the highest and counting down.
+pub fn apply_pagination_reverse<
```

Comment:
I'm confused about the name `reverse` given that it's asceding order by default?

Reply:
There's an existing `apply_pagination` which is the exact opposite of this. I could rename that to `apply_pagination_desc` and this one `apply_pagination_asc`. WDYT?

Reply:
Much better, yeah!